io.github.opentyrian.Opentyrian.yaml

```
@@ -0,0 +1,58 @@
+app-id: io.github.opentyrian.Opentyrian
```

https://opentyrian.github.io/ does not exist.
```suggestion
app-id: com.github.opentyrian.OpenTyrian
```

---

done!

---

well,

https://docs.flatpak.org/en/latest/conventions.html#application-ids

In the case of github it doesn't have to, it's based on the "path" of the repository and we prefer `.io`